analytics/views.py

```diff
@@ -402,6 +402,11 @@ def realm_summary_table(realm_minutes: Dict[str, float]) -> str:
         row['age_days'] = int((now - row['date_created']).total_seconds()
                               / 86400)
         row['is_new'] = row['age_days'] < 12 * 7
+        row['realm_admin_email'] = ', '.join(list(UserProfile.objects.filter(
+            realm=get_realm(
+                string_id=row['string_id']
+            ), is_realm_admin=True, is_active=True
+        ).values_list('email', flat=True)))
```

Comment:
This does database queries in a loop, and so is going to end up being really slow.  We should instead do a single database query that fetches all the realm admins (before this loop) and splice the data together in Python using the `realm_id` as the key.